### PR TITLE
A new Overview Board Concept

### PR DESCRIPTION
**Pull request by dhruv0000**
Signed-off-by: dhruv0000 <patel.4@iitj.ac.in>

**Description**
Trying out a different approach to the overview board.
Make sure you have some meshes running in the background and give this PR a spin at [http://localhost:3000/overview](http://localhost:3000/overview)

Secrrenshot1
overview2

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/OverviewComponent.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+          // Check if Linkerd data is present in the scan
+          if (Array.isArray(result.Linkerd)) {
+            const linkerdData = result.Linkerd.map(comp => {
+              const compData = {
+                name: self.generateMeshScanPodName(
+                  comp.metadata.name, 
+                  comp.metadata.labels["pod-template-hash"],
+                  comp.metadata.generateName
+                ),
+                component: comp.metadata.labels["linkerd.io/control-plane-component"],
```
Comment (leecalcote) [inline ui/components/OverviewComponent.js:117]: We need a single, uniform way of identifying that a mesh is installed. 

A proof point for whether we hit the mark in MeshSync v2. // @kumarabd @vineethvanga18

**Review by leecalcote — DISMISSED** [review]
Thanks, @dhruv0000. Merging as a currently unlinked view/page.

**Comment by leecalcote** [comment]
Tossing in a favicon improvement here, too.

**Comment by leecalcote** [comment]
@ramrodo in cases like this failed integration test - https://github.com/layer5io/meshery/pull/2242/checks?check_run_id=1725394009#step:13:179 - what do you recommend be done to troubleshoot?

**Comment by ramrodo** [comment]
> @ramrodo in cases like this failed integration test - https://github.com/layer5io/meshery/pull/2242/checks?check_run_id=1725394009#step:13:179 - what do you recommend be done to troubleshoot?

@leecalcote  Oh, I see what happens. On [PR#2150](https://github.com/layer5io/meshery/pull/2150/files), it was added another component with the same `data-cy` attribute:

https://github.com/layer5io/meshery/blob/master/ui/components/MeshConfigComponent.js#L543

We need to replace:

```
data-cy="chipContextName"
```

with another name, like:

```
data-cy="chipOperator"
```

**Comment by leecalcote** [comment]
Thank you, @ramrodo. I have changed this attribute value to avoid duplication. Rerunning checks..

**Pull request closed without merging**